packages/horizon-householder-appeal-publish/src/lib/logger.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Logger
+ *
+ * Creates a log instance with common configuration that can be
+ * used throughout the application
+ */
+
+const pino = require('pino');
+const config = require('../config');
+
+module.exports = pino({
```

Comment:
this is more of an FYI than a request to change - doing this makes life very difficult for writing unit tests in my experience. 

Not asking you to change - just flagging it as I know you love a good bit of jest ;)

Reply:
This is just the way it's done in other packages, I have little experience. Very happy to have suggestions for a different method.

Reply:
To clarify, this module exports an immediate function call to a third party package.

Writing tests for code that does this is very difficult. My comment was to suggest that for your own code, try to avoid doing this as it makes jest harder than it already is.

Reply:
OK